war/src/main/js/api/plugins.js

```diff
@@ -52,6 +55,14 @@ exports.availablePlugins = [
         ]
     },
     {
+        "category":"Pipelines and Continuous Delivery",
+        "plugins": [
+            { "name": "workflow-aggregator" },
+            { "name": "github-branch-source" },
+            { "name": "workflow-multibranch" }
```

Comment:
It worth to mention other pipeline plugins like build-pipeline and delivery-pipeline. Maybe promoted-builds also fits into the category. Otherwise the list seems to be too Pipeline Plugin specific